index.js

```diff
@@ -109,15 +77,16 @@ function FastForEach(spec) {
   this.afterAdd = spec.afterAdd;
   this.beforeRemove = spec.beforeRemove;
   this.templateNode = makeTemplateNode(
-    spec.name ? document.getElementById(spec.name).cloneNode(true) : spec.element
+    spec.templateNode || (spec.name ? document.getElementById(spec.name).cloneNode(true) : spec.element)
```

Comment:
When I refactored the test to instatiate the fastforeach programatically, I realized that I needed to add the template to the DOM. I didn't want to do that because it's nice if the tests don't have side-effects to each other, so I added this tiny option to specify the template node instead of its ID.

By the way, the element itself not always can be used as a template, for example when we use recursive templates.

Later on I've noticed that your previous tests were adding the template node to the DOM, so I definetely could have done the same.

If you don't like it please remove it and make the small refactoring, I'm very short on time these days.